zenrock/assetlist.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "zenrock",
+    "assets": [
+      {
+        "description": "The native token of Zenrock",
+        "denom_units": [
+          {
+            "denom": "urock",
+            "exponent": 0
+          },
+          {
+            "denom": "rock",
+            "exponent": 6
+          }
+        ],
+        "base": "urock",
+        "name": "Rock",
+        "display": "rock",
+        "symbol": "ROCK",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/zenrock/images/zenrock.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/zenrock/images/zenrock.svg"
+        },
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/zenrock/images/zenrock.png",
```

Comment:
this images aren't visually identical. Please use identical images, or place on of them into a new object within the images array.